telecine/src/main/AndroidManifest.xml

```diff
@@ -47,6 +47,16 @@
         android:taskAffinity=""
         android:exported="true"
         />
+
+    <service
+        android:icon="@drawable/ic_videocam_white_24dp"
+        android:label="@string/app_name"
```

Comment:
Done. Also found what might be a quick tiles bug. The system is caching the tile name for some period of time even after the app is updated. Have to remove and add it again.